felix/dataplane/linux/int_dataplane.go

```diff
@@ -904,6 +904,11 @@ func NewIntDataplaneDriver(config Config) *InternalDataplane {
 		}
 	}
 
+	var nftMaps nftables.MapsDataplane
+	if config.RulesConfig.NFTables {
+		nftMaps = nftablesV4RootTable.(nftables.MapsDataplane)
```

Comment:
Think I'd have `nftables.NewTable()` return the concrete type so you don't need to cast; not like  `nftablesV4RootTable` could be an iptables table, it's explicitly the nftables one.